tests/plugins/test_orchestrate_build.py

```diff
@@ -17,7 +17,9 @@
                                                             get_worker_build_info,
```

Comment:
Please refactor the build_orchestrate_build and test_orchestrate_build changes into a separate commit to make it a little easier to review.

Reply:
well it is whole one logical change, and it all depends on each other, as all has to be in place at the same time